api/v1alpha1/olsconfig_types.go

```diff
@@ -104,6 +104,9 @@ type OLSSpec struct {
 	// User data collection switches
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="User Data Collection"
 	UserDataCollection UserDataCollectionSpec `json:"userDataCollection,omitempty"`
+	// Additional CA for TLS communication
```

Comment:
`Additional CA certificates for TLS communication between OLS service and LLM Provider`  @raptorsun little more clear I believe.